.gitignore

```diff
@@ -0,0 +1,7 @@
+# Avoid directory
+**/__pycache__/
```

Comment:
You already ignore "*.pyc" pattern so it's not really needed to ignore the `__pycache__` directory.